src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -8,6 +8,7 @@
     <title>Add car</title>
 </head>
 <body>
+<%@include file="/WEB-INF/views/logout.jsp" %>
```

Comment:
You can write short link to the file:

<%@include file="../../logout.jsp" %>

Same for another